packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: System Log events for rate limits
+category: rate limits
+---
+
+# System Log events for rate limits
+
+Okta offers three standard System Log event types that you may encounter when Okta is enforcing [rate limits](/docs/reference/rate-limits/) on specific operations.
```

Comment:
I think we need to call out we have 3 different types, list the 3 out and then go into the details of operations. Then at the end we can link to the other 2 docs.

Reply:
@fireshot412

I have added a paragraph in the intro just below the operations information about the ops and client-based system log options.